src/gui/render/graphics_data/individual_int.hpp

```diff
@@ -0,0 +1,119 @@
+// -*- lsst-c++ -*-
+/*
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+ * more details.
+ */
+
+/**
+ * @file individual_int.hpp
+ *
+ * @author @AlemSnyder
+ *
+ * @brief Defines IndividualInt class
+ *
+ * @ingroup GUI  DATA_STRUCTURES
+ *
+ */
+
+//! This is a "data" class. Its purpose is to send data to the gpu for that
+// reason it should be moved into gui/data_structures
+// Also no namespace terrain
+
+#include "../../../entity/mesh.hpp"
+
+#include <GL/glew.h>
+#include <GLFW/glfw3.h>
+#include <glm/glm.hpp>
+
+#include <vector>
+
+#pragma once
+
+namespace gui {
+
+namespace data_structures {
+
+/**
+ * @brief Class to render non-instanced meshes
+ *
+ * @details Handles non-instanced meshes. Sends mesh data to GPU, and handles
+ * binding, and deleting data on GPU.
+ */
+class IndividualInt {
```

Comment:
WHAT is an individual integer, and what does it have to do with rendering meshes?

Please come up with a better name